rskj-core/src/main/java/co/rsk/scoring/PeerScoringManager.java

```diff
@@ -288,17 +332,31 @@ public PeerScoring getPeerScoring(InetAddress address) {
      * @param nodeID a node id
      */
     private void recordEventAndStartPunishment(PeerScoring peerScoring, EventType event, PunishmentCalculator punishmentCalculator, NodeID nodeID) {
-        peerScoring.recordEvent(event);
+        peerScoring.updateScoring(event);
 
         boolean shouldStartPunishment = !scoringCalculator.hasGoodReputation(peerScoring) && peerScoring.hasGoodReputation();
         if (shouldStartPunishment) {
             long punishmentTime = punishmentCalculator.calculate(peerScoring.getPunishmentCounter(), peerScoring.getScore());
             peerScoring.startPunishment(punishmentTime);
 
-            String nodeIDFormated = nodeIdForLog(nodeID);
-            logger.debug("NodeID {} has been punished for {} milliseconds. Last event {}", nodeIDFormated, punishmentTime, event);
-            logger.debug("{}", PeerScoringInformation.buildByScoring(peerScoring, nodeIDFormated, ""));
+            String nodeIDFormatted = nodeIdForLog(nodeID);
+            logger.debug("NodeID {} has been punished for {} milliseconds. Last event {}", nodeIDFormatted, punishmentTime, event);
+            logger.debug("{}", PeerScoringInformation.buildByScoring(peerScoring, nodeIDFormatted, ""));
+        }
+    }
+
+    private void logRecordedEvent(NodeID id, InetAddress address, EventType event, String message, Object[] messageArgs) {
+        String completeMessage = "Recorded {} from node [{}]";

```

Comment:
can we check for `logger.isDebugEnabled()` and skip building this message if it's `false`?

Reply:
Done